resources.js

```diff
@@ -142,7 +143,7 @@ function loadResource(name,max,rate,tradable,stackable,color) {
     color = color || 'info';
     if (!global['resource'][name]){
         global['resource'][name] = {
-            name: name === 'Money' ? '$' : name,
+            name: name === races[global.race.species].name? name : name === 'Money' ? '$' : loc('resource_' + name + '_name'),
```

Comment:
What's going on here with the races? race names aren't related to resource names.